### DISCUSSION TITLE
specific starting and end time

### DISCUSSION THREAD

**Discussion by KlausV66** [dbuezas/lovelace-plotly-graph-card#346] [Show and tell]
Hello,
pretty nice graph possibilities, but I didn't find any possibilities to use a fix time start and end, like 8am - 5pm. The best case would be to have it by date and sunrise / sunset. I need it for my solar graph. 
Thanks for the input.
Klaus

**Comment by dbuezas** [comment]
There's no specific feature for that, but you can achieve it with a tiny bit of code. See this example: https://github.com/dbuezas/lovelace-plotly-graph-card/discussions/220#discussioncomment-4832891

**Comment by KlausV66** [comment]
Thanks, but not so easy to understandy quickly. The range of 24h seams easier to handle as a fix range of start and end point. 
Anyway, I will study it.
